target/linux/sunxi/base-files/etc/board.d/02_network

```diff
@@ -18,6 +18,9 @@ case "$(board_name)" in
 "xunlong,orangepi-r1")
 	ucidef_set_interfaces_lan_wan "eth0" "eth1"
 	;;
+"friendlyarm,nanopi-r1")
+	ucidef_set_interfaces_lan_wan "eth1" "eth0"
+	;;
```

Comment:
Please apply alphabetic sorting (I know it hasn't been before, so maybe just correct the file now.